flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/aggregate/ProcTimeUnboundedOver.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.runtime.aggregate;
+
+import org.apache.flink.api.common.state.ValueState;
+import org.apache.flink.api.common.state.ValueStateDescriptor;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.streaming.api.functions.KeyedProcessFunction;
+import org.apache.flink.table.api.TableConfig;
+import org.apache.flink.table.dataformat.BaseRow;
+import org.apache.flink.table.dataformat.JoinedRow;
+import org.apache.flink.table.dataview.PerKeyStateDataViewStore;
+import org.apache.flink.table.generated.AggsHandleFunction;
+import org.apache.flink.table.generated.GeneratedAggsHandleFunction;
+import org.apache.flink.table.type.InternalType;
+import org.apache.flink.table.typeutils.BaseRowTypeInfo;
+import org.apache.flink.util.Collector;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Process Function for processing-time unbounded OVER window.
+ */
+public class ProcTimeUnboundedOver<K>
+	extends ProcessFunctionWithCleanupState<K, BaseRow, BaseRow> {
+	private static final Logger LOG = LoggerFactory.getLogger(ProcTimeUnboundedOver.class);
+
+	private GeneratedAggsHandleFunction genAggsHandler;
+	private InternalType[] accTypes;
+
+	private AggsHandleFunction function;
+	private ValueState<BaseRow> accState;
+	private JoinedRow output;
+
+	public ProcTimeUnboundedOver(
+		GeneratedAggsHandleFunction genAggsHandler,
+		InternalType[] accTypes,
+		TableConfig tableConfig) {
+		super(tableConfig);
+		this.genAggsHandler = genAggsHandler;
+		this.accTypes = accTypes;
+	}
+
+	@Override
+	public void open(Configuration parameters) throws Exception {
+		LOG.debug("Compiling AggregateHelper: " + genAggsHandler.getClassName() + " \n\n" +
```

Comment:
Actually, we don't need this anymore, because `genAggsHandler.newInstance` will log the code.